src/index.html

```diff
@@ -5,8 +5,465 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <img class="header__img" src ="images/logo.png"
+          alt="logo"href="#home">
+        <nav class="nav">
+          <ul class="nav__list"></ul>
+          <li class="nav__li-item">
+            <a
+              class="nav__link" href="#video"
+            >
+              Video
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="nav__link" href="#photo"
```

Comment:
if you have two or more attributes , move them on new lines, in your case, leave them on one line